apps/web-giddh/src/app/bank-statement/components/stepper-form/stepper-form.component.ts

```
@@ -0,0 +1,613 @@
+        });
+    }
+
+    /**
+     * Validates and proceeds to next step
+     * 
+     * @param {MatStepper} stepper - Material stepper reference
+     * @param {FormGroup} currentForm - Current step form
+     * @memberof StepperFormComponent
+     */
+    public nextStep(stepper: MatStepper, currentForm: FormGroup, stepNumber: number): void {
+        debugger;
```

Biome (2.1.2)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)